src/main/java/org/apache/sling/distribution/queue/DistributionQueueItem.java

```diff
@@ -44,7 +47,7 @@ public DistributionQueueItem(String packageId, long size, Map<String, Object> ba
         super(base);
         this.packageId = packageId;
         this.size = size;
-
+        this.base = base;
```

Comment:
I think we shouldn't keep this field because
1. it is unnecessary
2. it is possible to invoke `.put` directly on `this` because it extends `ValueMapDecorator`, which can make `base` out of sync with what's there in `DistributionQueueItem` object